### ISSUE TITLE
Update page <title> text

### ISSUE THREAD

**Issue by alundgard** [sul-dlss/heracles-etd#190]
Currently, the `<title>` is set to "Heracles Etd".
Update this element to match the page's `<h1>` in the masthead.
That is:

- `<title>Submit your dissertation or thesis</title>` for the student-facing submission pages.
- `<title>View dissertation or thesis</title>` for the faculty-facing reader review page.

### Reported by SODA

> [Summary]
> Page TITLE element does not identify purpose of page
> 
> [Description]
> The TITLE element on this page does not identify the contents or purpose of the page.
> A page title of "Heracles Etd" is used which is not informative. 
> 
> [Recommended to fix]
> Rule : 
> The page `<title>` MUST be accurate and informative. The page `<title>` SHOULD be unique, if possible.
> 
> How To Fix: 
> Fix this issue by using the `<title>` element in the document `<head>` to provide an accurate, informative, and (preferably) unique page title.
> 
> ```
> <html lang="en">
> <head>
> <title>Products and Services</title>
> </head>
> 
> ```
> Reference: 
> Deque University: https://dequeuniversity.com/class/semantic-structure/page-title/meaningful-page-title
> 
> Background: 
> People who are blind cannot visually scan a page for a main heading or similar feature to quickly determine the purpose of a particular page. The <title> element is the first thing a screen reader reads when loading a page or switching to a new browser tab. A descriptive <title> element gives screen reader users a quick overview of a page without having to read the page itself.
